docker/Dockerfile

```diff
@@ -1,43 +1,45 @@
 FROM rust:1.84 AS orion-builder
 
-#RUN rustup component add rustfmt
+RUN apt-get update && \
+    apt-get install -y protobuf-compiler && \
+    rm -rf /var/lib/apt/lists/*
 
 WORKDIR /tmp/orion
 
-RUN <<EOF
-apt update
-apt install -y protobuf-compiler
-EOF
+COPY Cargo.toml Cargo.lock ./
 
+RUN mkdir -p src && echo 'fn main() {}' > src/main.rs
 
-COPY ./orion-xds ./orion-xds
-COPY ./orion-proxy ./orion-proxy
-COPY ./orion-data-plane-api ./orion-data-plane-api
-COPY ./orion-error ./orion-error
-COPY ./orion-lib ./orion-lib
-COPY ./envoy-data-plane-api ./envoy-data-plane-api
-COPY ./orion-configuration ./orion-configuration
+RUN set -e; \
+    for member in \
+        orion-lib \
+        orion-xds \
+        orion-proxy \
+        orion-error \
+        orion-configuration \
+        orion-data-plane-api \
+        envoy-data-plane-api; \
+    do \
+        mkdir -p "${member}/src"; \
+        printf '[package]\nname = "%s"\nversion = "0.1.0"\nedition = "2021"\n\n[lib]\npath = "src/lib.rs"\n' \
+        "${member}" > "${member}/Cargo.toml"; \
+        touch "${member}/src/lib.rs"; \
+    done
```

Comment:
Why construct `src/main.rs` and many `Cargo.toml`, `lib.rs` here? Any thing wrong with the code before?